tests/unit/src/playwright-runner.spec.js

```diff
@@ -83,8 +83,6 @@ describe('playwright-runner', function () {
         path.join(MOCK_CWD, 'sauce.config.js'),
         '--timeout',
         1800000,
-        '--browser',
-        'chromium',
```

Comment:
Would it be possible we have a test with and without project support ?
I fear that if we dont, it may bite us later.